API/config.example.json

```
@@ -5,10 +5,10 @@
     "database": {
       "host": "localhost",
       "port": 5432,
-      "username": "",
-      "password": "",
-      "name": "risk",
+      "username": "postgres",
+      "password": "Bodensjr",
+      "name": "Ocat",
```

```suggestion
      "username": "",
      "password": "",
      "name": "",
```
I would recommend leaving these empty.  You don't want to publish passwords / usernames to a public cloud service.